### ISSUE TITLE
Aditional options

### ISSUE THREAD

**Issue by alagos** [testdouble/gem_dating#9]
Great gem. I was thinking that would be good to allow:
- sort gems by date 
- output in json/yaml/whatever format
- hide the "Unknown" gems... or adding them in another json/yaml key, in case that those formats are supported

**Comment by brandonzylstra** [comment]
Yes, sorting by date is **_by far_** the most useful way to sort the output.  The current sorting creates an impediment to using the information since we must scan the whole column and try to remember which gems are oldest and most in need of looking for updates.

**Comment by Daniel-N-Huss** [comment]
Great suggestions! I've [released these in 0.1.2](https://rubygems.org/gems/gem_dating/versions/0.1.2).
